mypy/server/aststrip.py

```diff
@@ -233,6 +228,11 @@ def visit_import_all(self, node: ImportAll) -> None:
                 del self.names[name]
         node.imported_names = []
 
+    def visit_for_stmt(self, node: ForStmt) -> None:
```

Comment:
It looks like we don't have `visit_with_stmt` while it also might need to be stripped (at least for consistency).

Reply:
Hm probably `target_type` needs to get cleared. I'll look at it for a different PR